src/model/spec/userProfile.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+import * as deepFreeze from 'deep-freeze';
+import { UserProfile } from '../userProfile'
+import {} from 'mocha'
+import {} from 'core-js'
+
+describe('userProfile', () => {
+  let userProfile = null;
+
+  beforeEach(() => {
+    userProfile = new UserProfile();
+  })
```

Comment:
@franmolmedo , I have Updated the test to use _beforeEach_, is it ok how is being used? On the other hand the comment where you commented using enzyme mount/shallow/render is lost in the PR (because of the last commit), I had use this to instantiate react components (mount and shallow), but never tried to instantiate an object usign this approach, what could be the advantages? Thanks